tests/framework/extensions/scalinginput/config.go

```diff
@@ -12,4 +15,7 @@ const (
 type Config struct {
 	NodesAndRoles     *machinepools.NodeRoles `json:"nodesAndRoles" yaml:"nodesAndRoles"`
 	NodesAndRolesRKE1 *nodepools.NodeRoles    `json:"nodesAndRolesRKE1" yaml:"nodesAndRolesRKE1"`
+	AKSNodePool       *aks.NodePool           `json:"aksNodePool" yaml:"aksNodePool"`
+	EKSNodeGroup      *eks.NodeGroupConfig    `json:"eksNodeGroup" yaml:"eksNodeGroup"`
```

Comment:
So I understand why EKS is not named like the others, but config-wise, should we unify this one with other hosted tags?

Reply:
I don't have a strong opposition to it, I just named each hosted cluster to what their respective cloud provider calls it. EKS is nodegroup and AKS/GKE refer to it as node pool.

Reply:
Changed it to NodePool just to match across the board.